erpnext/controllers/accounts_controller.py

```diff
@@ -1838,6 +1838,17 @@ def create_advance_and_reconcile(self, party_link):
 		jv.save()
 		jv.submit()
 
+	def check_conversion_rate(self):
+		default_currency = erpnext.get_company_currency(self.company)
+		if not default_currency:
+			throw(_("Please enter default currency in Company Master"))
+		if (
+			(self.currency == default_currency and flt(self.conversion_rate) != 1.00)
+			or not self.conversion_rate
+			or (self.currency != default_currency and flt(self.conversion_rate) == 1.00)
```

Comment:
Conversion rate between few currencies can actually be 1.0 (e.g. pegged currencies)

Lets avoid validating that (or maybe just give warning instead)